### DISCUSSION TITLE
Question about Volumtric Fraction

### DISCUSSION THREAD

**Discussion by Calvinsgithub** [OpenRadioss/OpenRadioss#2804] [Q&A]
Dear @MarianBulla ,
I hope you are doing well.

Thank you for developing the excellent software "OpenRadioss.".I'm new to this app, and I have a question for you. Recently, I simulated metal cutting using the cel method under the Radioss plate.  I used type 18 contact type and material with LAW51, and the metal EOS option was POLYNOMIAL.Volumtric Fraction is shown in the Hyperview calculation results.Why does the material volume behind the tool enter the upper air grid area 1 after being cut by the tool? (The blue part in the picture represents the metal material Alu, and the red part represents the air.)？

![`0DUD4XYUO`8V6M0`%P(9N9](https://github.com/OpenRadioss/OpenRadioss/assets/142204691/067764ae-9b28-4fa3-8b19-a2791a0e97a8)

Best regards,
Calvins

**Comment by MarianBulla** [comment]
Dear @Calvinsgithub,
thank you very much for your interest in OpenRadioss.
Just a small clarification: OpenRadioss was (and still is beeing further developed and maintained) developed by a large number of developers and now with the help of the OpenRadioss community, so it is a team offort :-) 
Regarding your question: Can you add your model here, please ? 
It's hard to judge without having the model....beeing blind.

Thanks and best regards,
Marian

**Reply by Calvinsgithub** [reply]
Dear @MarianBulla 

Thank you for your support and kindness.I added the model file. This is a model for metal cutting simulation using CEL method, but the performance of Volumtric Fraction in the calculation results is abnormal. Due to insufficient familiarity with the software, I hope to get your guidance.

Thank you in advance for your assistance.

[cel_cutting.zip](https://github.com/user-attachments/files/16172611/cel_cutting.zip)

Best regards
Calvins
